src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,74 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React, { useMemo } from 'react';
+import { breadcrumbStyles, breadcrumbLinkStyles } from './Breadcrumb.style';
+import useTheme from 'hooks/useTheme';
+import BreadcrumbItem from './BreadcrumbItem/BreadcrumbItem';
+import isEmpty from 'lodash/isEmpty';
+import uniqueId from 'lodash/uniqueId';
+import { Link } from 'react-router-dom';
+import BreadcrumbCollapsed from './BreadcrumbCollapsed/BreadcrumbCollapsed';
+
+export type Props = {
+  separatorContent?: '*' | '>' | '/';
+  data: [];
```

Comment:
define data so you can use it as `BreadcrumbItemData[]`

e.g 
```
type BreadcrumbItemData = {
  to: string;
  label: string;
}
```

OPTIONAL: data literal not needed as these types are models so `BreadcrumbItem` should do the trick